nodejs/week1/src/backend/server.js

```diff
@@ -0,0 +1,38 @@
+const express = require("express");
+const app = express();
+const port = 3000;
+
+app.get("/", (request, response) => {
+  response.send("Meal Sharing Web App");
+});
+
+const server = app.listen(port, () => {
+  console.log(`Example app listening at http://localhost:${port}`);
+});
```

Comment:
It's not a technical requirement, but it's a convention to place this bit at the end of the file. 🙂 

Also, `port` ➡️  `PORT` because the convention is to capitalize constants that are primitive (string, number, boolean..., not object or array) - see [here](https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/).